src/ucp/api/ucp.h

```diff
@@ -333,6 +335,19 @@ typedef enum {
 
 /**
  * @ingroup UCP_DATATYPE
+ * @brief Generate a reusable identifier for Scatter-gather IOV data type.
```

Comment:
1. clear and simple API is more important
2. using ucp_dt_create_struct() would create a handle, which can contain the cached memory keys.